code/modules/mob/living/carbon/xenomorph/hive_datum.dm

```diff
@@ -727,12 +727,16 @@ to_chat will check for valid clients itself already so no need to double check f
 		if(xeno_job.total_positions < (-difference + xeno_job.current_positions))
 			xeno_job.set_job_positions(-difference + xeno_job.current_positions)
 	for(var/obj/structure/resin/silo/silo as() in GLOB.xeno_resin_silos)
-		if(isalamoarea(get_area(silo)))
-			continue
 		if(!is_ground_level(silo.z))
 			continue
 		qdel(silo)
 
+	SSpoints.xeno_points_by_hive[hivenumber] = SILO_PRICE //Give a free silo when going shipside
```

Comment:
shoulnd't this just add ? otherwise it would reduce points if the xenos had more, but if you intend that, its okay.
```suggestion
	SSpoints.xeno_points_by_hive[hivenumber] += SILO_PRICE //Give a free silo when going shipside
```

Reply:
Yeah it's intended,  because the silo shipside is stronger